graphql-client/src/Data/GraphQL/Error.hs

```diff
@@ -30,13 +33,31 @@ data GraphQLError = GraphQLError
   }
   deriving (Show, Eq, Generic, ToJSON, FromJSON)
 
+instance Exception GraphQLError where
```

Comment:
Does this need Exception? It seems like you just need to override displayException in GraphQLException. I'd prefer if this didnt have Exception, to avoid confusion over what's actually being thrown

Reply:
Good point, done.